pkg/volume/gce_pd/gce_util.go

```diff
@@ -149,11 +150,60 @@ func (mounter *gceSafeFormatAndMount) Mount(source string, target string, fstype
 	}
 	args = append(args, options...)
 	args = append(args, source, target)
-	glog.V(5).Infof("exec-ing: /usr/share/google/safe_format_and_mount %v", args)
-	cmd := mounter.runner.Command("/usr/share/google/safe_format_and_mount", args...)
+	safeFormatAndMountPath := "/usr/share/google/safe_format_and_mount"
+	glog.V(5).Infof("exec-ing: %s %v", safeFormatAndMountPath, args)
+	cmd := mounter.runner.Command(safeFormatAndMountPath, args...)
 	dataOut, err := cmd.CombinedOutput()
 	if err != nil {
-		glog.V(5).Infof("error running /usr/share/google/safe_format_and_mount\n%s", string(dataOut))
+		glog.V(5).Infof("error running %s\n%s", safeFormatAndMountPath, string(dataOut))
+		// if safe_format_and_mount is not available on this system use linux utilities directly.
+		if _, error := os.Stat(safeFormatAndMountPath); os.IsNotExist(error) {
+			return mounter.formatAndMount(source, target, fstype, options)
+		}
+	}
+	return err
+}
+
+// formatAndMount uses unix utils to format and mount the given disk
+func (mounter *gceSafeFormatAndMount) formatAndMount(source string, target string, fstype string, options []string) error {
+
+	// Add the default options from safe_format_and_mount
+	options = append(options, "defaults")
+
+	// Try to mount the disk. Let 'mount' try to determine the fstype; if it determines the
+	// filesystem type to be "null" then assume the disk is unformatted.
+	err := mounter.Interface.Mount(source, target, "", options)
+	if err != nil && strings.Contains(err.Error(), "unknown filesystem type '(null)'") {
+		// It is very likely that this disk is not formatted. Double check using 'file'
+		var notFormatted bool
+		notFormatted, err = mounter.diskLooksUnformatted(source)
```

Comment:
Optional, maybe a TODO: Would be nice to check that the device is not partitioned.  e.g. if I say /dev/sdb and I meant sdb1, it should fail rather than blow away my data.